OpenRA.Game/FileFormats/WavLoader.cs

```diff
@@ -57,8 +56,8 @@ bool LoadSound(Stream s)
 						AudioFormat = s.ReadInt16();
 						Type = (WaveType)AudioFormat;
 
-						if (Type != WaveType.Pcm && Type != WaveType.ImaAdpcm)
-							throw new NotSupportedException("Compression type is not supported.");
+						if (!Enum.IsDefined(typeof(WaveType), Type))
+							throw new NotSupportedException("Compression type {0} is not supported.".F(AudioFormat));
```

Comment:
`AudioFormat` is *not* the compression type, so this message does not make sense.